### PR TITLE
Updated readmes and contributing docs

### PR DESCRIPTION
**Pull request by gregorypratt**

### CODE REVIEW

**Comment by cognitom** [comment]
@gregorypratt oh, my great English teacher! Thanks, a lot!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,31 +1,37 @@
 # Contributing to Riot Examples
 
-If you’d like to contribute a feature or bug fix, you can fork Riot, commit your changes, & send a pull request. Please make sure to search the issue tracker first.
+If you'd like to contribute an example or bug fix, feel free to send us a pull
+request. Please make sure to search the issue tracker first for duplicates.
```
Comment (cognitom) [inline CONTRIBUTING.md]: It seems better without `new line`?
GitHub Markdown replaces one with `<br>`, two with `<p>`.
https://help.github.com/articles/writing-on-github/#newlines

File: README.md [hunk 2]
```diff
@@ -1,39 +1,31 @@
...
-We have several examples for the beginners. These are relatively well tested by users and collaborators.
+We have several examples to choose from; each one demonstrates different
+aspects Riot:
 
 Source | Demo
 :-- | :--
-[Todo App](todo-app) | [try & edit](http://riotjs.com/examples/plunker/?app=todo-app)
-[Todo App (pre-compiled)](todo-app-precompiled) | [try](http://riotjs.com/examples/todo-app-precompiled/)
+[Todo](todo-app) | [Plunker](http://riotjs.com/examples/plunker/?app=todo-app)
+[Todo (pre-compiled)](todo-app-precompiled) | [Plunker](http://riotjs.com/examples/todo-app-precompiled/)
```
Comment (cognitom) [inline README.md]: This is not linked to Plunker. The label should be just "demo" or something?

**Comment by gregorypratt** [comment]
Haha! I can only speak one language though!

**Comment by cognitom** [comment]
:smile:

**Pull request merged**